modules/python/python-plugin.c

```diff
@@ -64,6 +64,8 @@ _py_init_interpreter(void)
       python_debugger_append_inittab();
 
       Py_Initialize();
+      char *argv[] = {"syslog-ng"};
+      py_setargv(1, argv);
```

Comment:
As `py_setargv` is only ever used here (and in the tests), and only with `argv` having a single `syslog-ng` element, wouldn't it be better if the whole dance of making `py_setargv` work the same across python2 and python3 was dropped?

And instead, introduced something like `void py_init_argv(void)`, that would just set it to syslog-ng, without all of the things [this](https://github.com/balabit/syslog-ng/pull/1754/files#diff-3aa4084fce6fac1998206998e1793f19R39) does.

Reply:
Off the top of my head, I'm thinking something along these lines:

```c
// compat-python-v2
void 
py_init_argv(void) {
  static char *argv[] = {"syslog-ng"};
  PySys_SetArgvEx(1, argv, 0);
}

// compat-python-v3
void 
py_init_argv(void) {
  static wchar_t *argv[] = {L"syslog-ng"};
  PySys_SetArgvEx(1, argv, 0);
}
```

Reply:
Thanks for the comment, especially with the suggested code. Will do this way!